### DISCUSSION TITLE
Authorized addresses != mailboxes

### DISCUSSION THREAD

**Discussion by IanNorris** [simple-login/app#1020] [Feature Requests]
I added my work email address as an authorized address for one of my mailboxes so that I could send to a reverse alias when I was on my work machine and have it send as from my personal account (this is a really handy feature btw), but I noticed I can no longer email myself from my work account because it thinks I've emailed my own mailbox!

I often email myself notes, so unfortunately this breaks my ability to do this. Can the authorized addresses not count as mailboxes please so emails to them go through as normal? It'd be great to have the ability to do both.

**Comment by IanNorris** [comment]
Just a follow up, was the behavior of this recently changed to send an access denied error instead of just silently discarding the email?

I spent close to 4 hours today trying to figure out why I was getting errors when sending from my work email (Exchange Online) to one of my other domains, but not the one I'd previously linked.

When an email bounces in this way can we get an entry in the portal to help track this down a bit easier? Ideally an entry under the alias with the sender and a a "You can't email yourself" message or something. Obviously I'd prefer if it just worked, but I'll settle for a gentle reminder each time I try!

**Reply by nguyenkims** [reply]
I created this PR https://github.com/simple-login/app/pull/1035 to make sure user is informed every time a "cycle" email happens. It'll be deployed in the next version.
